deployments/sgx_aesmd/base/sgx_default_qcnl.conf

```diff
@@ -0,0 +1,2 @@
+PCCS_URL=https://localhost:8081/sgx/certification/v2/
```

Comment:
Can you use kustomize to set these values too, or is it limited just to YAML? Would be nice if this could be the "localhost" overlay.

Reply:
true. I'll see what I can find. It already looks `sgx_default_qcnl.conf` does not necessarily need to be a file but can be generated from configmap literals...

Reply:
I played with this a bit. I was not able to get it with kustomize directly. However, it is possible to build an overlay with custom settings like this:
```
bases:
- ../../base
generatorOptions:
  disableNameSuffixHash: true
configMapGenerator:
- name: sgx-attestation-conf
  behavior: merge
  files:
  - sgx_default_qcnl.conf
```

Reply:
If qpl would allow to work with env variables, setting PCCS_URL without .conf file would be easy with kustomize `configMapGenerator`